### ISSUE TITLE
Flashlights

### ISSUE THREAD

**Issue by captainblobfish** [yogstation13/yogstation#1231]
I've noticed a few times that sometimes, after putting a flashlight on my taser, or energy gun, I will turn it on, run into a dark place, run out, and run through another unlit place, and the flashlight (which is still on) will stop illuminating the area. I have to turn the flashlight off and on to illuminate the area again. I'm sure this is a bug of some sort, and I thank everyone for their time. I know this isn't a major issue, it's just a slight annoyance, but I hope you guys have time at some point to fix it. Or it may just be a problem on my end. Maybe I'm the only one with this problem?

**Comment by Time-Green** [comment]
This happens with alot of light sources

**Comment by captainblobfish** [comment]
Alright, so it's not just with me. That's great to know. I mean, currently it's not THAT much of a problem, considering all you have to do is turn the flashlight off and on.

**Comment by aphielanis** [comment]
Yup its a bug that's been around for ever. It seems to happen when removing from self or picking up a active light source.

**Comment by Dragonforcefive** [comment]
Has no one else reported this?

**Comment by Time-Green** [comment]
Alot of people have it, but it's so common people got used to it and forgot about it.
